### PR TITLE
Chinese translation improvements

### PR DESCRIPTION
**Pull request by St-Maxwell**
@zoziha Please review the changes.

### CODE REVIEW

File: locale/zh_CN/LC_MESSAGES/how-to.po [hunk 1]
```diff
@@ -19,7 +19,7 @@ msgstr ""
 
 #: ../../pages/how-to/index.md:3
 msgid "How-To guides"
-msgstr "操作指南"
+msgstr "操作指引"
```
Comment (zoziha) [inline locale/zh_CN/LC_MESSAGES/how-to.po:22]: I think the "指南" is better, the "指引" is more verbs (动词).

File: locale/zh_CN/LC_MESSAGES/tutorial.po [hunk 2]
```diff
@@ -124,25 +124,25 @@ msgid ""
...
+"像测试框架这样的开发依赖关系，我们通过指定标记（*tag*）来选择严格的版本引脚。"
 
 #: ../../pages/tutorial/dependencies.md:94
 msgid ""
 "Now we can write a simple unit test, since our function works with units,"
 " we will create scratch units to create the input and capture the output."
 " For now we will add a simple one line substitution as single test case"
 msgstr ""
-"现在我们可以编写一个简单的单元测试，因为我们的函数与单元一起工作，我们将创建暂存单元来创建输入并捕获输出。"
+"现在我们可以编写一个简单的单元测试。因为我们的函数执行需要文件通道，我们将创建scratch通道来创建输入并捕获输出。"
```
Comment (zoziha) [inline locale/zh_CN/LC_MESSAGES/tutorial.po:145]:
```suggestion
"现在我们可以编写一个简单的单元测试。因为我们的函数执行需要文件通道，我们将创建暂存（scratch）通道来创建输入并捕获输出。"
```

File: locale/zh_CN/LC_MESSAGES/tutorial.po [hunk 3]
```diff
@@ -160,20 +160,20 @@ msgid ""
 "this kind of tasks can usually be done in a separate procedure and reused"
 " in several tests."
 msgstr ""
-"为多个单元测试创建暂存单元将是重复的，这种任务通常可以在单独的过程中完成，并在多个测试中重用。"
+"在多个单元测试中创建scratch通道比较繁琐，通常会在一个单独的过程中创建它们，然后在多个测试中重用。"
```
Comment (zoziha) [inline locale/zh_CN/LC_MESSAGES/tutorial.po:163]: Ditto.

It may be better to provide a Chinese translation of "scratch".

**Review by zoziha — APPROVED** [review]
LGTM

File: locale/zh_CN/LC_MESSAGES/index.po [hunk 4]
```diff
@@ -38,18 +38,18 @@ msgstr "Fortran包管理器"
...
 #: ../../pages/index.md:2
 msgid "Package manager and build system for Fortran"
-msgstr "适用Fortran的包管理器和构建系统"
+msgstr "Fortran的包管理器与构建系统"
 
 #: ../../pages/index.md:19
 msgid ""
 "Welcome to the documentation for the Fortran Package Manager (fpm)."
-msgstr "欢迎来到Fortran包管理器（fpm）的文档。"
+msgstr "这里是Fortran包管理器（Fortran Package Manager）的文档。"
```
[the remark this answers is not in the document]
Comment (St-Maxwell) [reply locale/zh_CN/LC_MESSAGES/index.po:46]: Keep "欢迎" is OK, but I would prefer not to translate this sentence so directly.

**Pull request merged**